pkg/registry/cache_test.go

```diff
@@ -0,0 +1,37 @@
+package registry_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/aws/eks-anywhere/pkg/registry"
+)
+
+func TestCache_Get(t *testing.T) {
+	cache := registry.NewCache()
+
+	result, err := cache.Get("localhost", "", false)
+
+	assert.NoError(t, err)
+	ociRegistry, ok := result.(*registry.OCIRegistryClient)
+	assert.True(t, ok)
+	assert.Equal(t, "localhost", ociRegistry.GetHost())
+
+	result, err = cache.Get("localhost", "", false)
+
+	assert.NoError(t, err)
```

Comment:
```suggestion
	result, err = cache.Get("localhost", "", false)
	assert.NoError(t, err)	
```
For improved readability, assuming you agree.

Reply:
Hmm, this was spaced this way for given/when/then, but this isn't a great example of that. Let me think about it.

Reply:
Yeh, I think no space makes sense since this test is covering 3 different test cases. Fixed.